import-automation/import-progress-dashboard-frontend/client/src/App.css

```diff
@@ -0,0 +1,38 @@
+.App {
```

Comment:
you might not need any of this css anymore

Reply:
removed